src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/DefaultVSCredentialServiceProvider.cs

```diff
@@ -20,41 +22,38 @@ public class DefaultVSCredentialServiceProvider : ICredentialServiceProvider
     {
 
         private readonly Lazy<INuGetUILogger> _outputConsoleLogger;
-        private readonly IServiceProvider _serviceProvider;
+        private readonly IAsyncServiceProvider _asyncServiceProvider;
 
         [ImportingConstructor]
+        internal DefaultVSCredentialServiceProvider(Lazy<INuGetUILogger> outputConsoleLogger)
+            : this(AsyncServiceProvider.GlobalProvider, outputConsoleLogger)
+        { }
+
         internal DefaultVSCredentialServiceProvider(
-           [Import(typeof(SVsServiceProvider))]
-            IServiceProvider serviceProvider,
-           Lazy<INuGetUILogger> outputConsoleLogger
+            IAsyncServiceProvider asyncServiceProvider,
+            Lazy<INuGetUILogger> outputConsoleLogger
```

Comment:
is this still needed? are we using it in tests?

Reply:
not really needed now, just keep this pattern, in case we need to pass different AsyncServiceProvider in the future.